portal/src/main/webapp/js/src/study-view/component/ScatterPlots.js

```diff
@@ -428,14 +435,22 @@ var ScatterPlots = function() {
                     arr_id: d.case_id,
                     transform: "translate(" + elem.xScale(d.x_val) + ", " + elem.yScale(d.y_val) + ")",
                     fill: fill,
-                    stroke: stroke
-                });
-            })
-            .attr("d", d3.svg.symbol()
-                .size(style.size)
-                .type(style.shape))
-            .attr("stroke-width", style.stroke_width)
-            .attr("z-index", "100");
+                    stroke: stroke,
```

Comment:
FYI - in other performance tests we made, we found that stroke attribute slows down the plot. You could consider not adding it by default and only add it when the circle has been clicked/selected.